checkov/bicep/checks/resource/base_registry.py

```diff
@@ -6,6 +6,8 @@
 from checkov.common.checks.base_check_registry import BaseCheckRegistry
 from checkov.runner_filter import RunnerFilter
 
+GRAPH_CHECK_IDS = ("CKV_AZURE_23",)
```

Comment:
I was hesitant to create a real solution for this, because it is a special case, there is a Python ARM check with this number and I don't want it to be added to the bicep registry. But I will check, if I can come up with something nicer 🙂

Reply:
Suggestion - when removing arm checks, also check the bicep graph registry :)